automotive/odCANDataStructureGenerator/org.opendavinci.candatamodel/src/org/opendavinci/CANDataModel.xtext

```diff
@@ -24,36 +24,39 @@ grammar org.opendavinci.CANDataModel
 generate canDataModel "http://www.opendavinci.org/CANDataModel"
 
 Model:
-    odvdFiles         += ODVDFile+
-    cansignals        += CANSignal+
-    cansignalmappings += CANSignalMapping*
-    cansignaltestings += CANSignalTesting*;
+    odvdFiles           += ODVDFile+
+    canmessages         += CANMessage+
+    canmessagemappings  += CANMessageMapping*
+    canmessagetestings  += CANMessageTesting*;
 
 ODVDFile:
     'using' odvdFileName = ID (';'|'.odvd' ';');
 
+CANMessage: 
+    ('CAN Message'|'CAN message') canIdentifier = HEXNUMBER canMessageName = ValidIdentifier ':' canMessageLength = FLOATNUMBER '{'
```

Comment:
@fgiaimo Could you please improve the language design slightly (cf. OpenDLV.core):

"CAN Message myMessage : 8 {"
--> 
"CANMessage myMessage is 8 bytes {

Then, the content is more intuitive at the first glance.